
# Self-check Python Skill

# Task

Clone this repo. Do

```bash
cd PyDebug
python run.py
```

Observe the error and try to debug using `ipdb` and fix the issue.
